### PR TITLE
Fix linting issues following typescript-eslint upgrade

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

`typescript-eslint` was upgraded and apparently introduced a number of rules into the recommended set.

This PR addresses those new violations.

## Complexity

Complexity: Low

### CODE REVIEW

**Comment by darrellwarde** [comment]
@tbwiss I had to make [this](https://github.com/neo4j/graphql/pull/3647/files#diff-1c1e6fa933d29a0e7132e1b4dd46bbc91f022118c183713d8eb902483eb8ec40) change in the Toolbox, because it was claiming that the `Text` type would stringify to "[Object object]". I noticed `Text` implemented `Iterable`, so converted into an array and stringified that.

**Comment by neo4j-team-graphql** [comment]
# Performance Report

No Performance Changes


<details>
    <summary>Show Full Table</summary>
    
| name | dbHits | old dbHits | time (ms) | old time (ms) | maxRows |
| ---- | ------ | ---------- | --------- | ------------- | ------- |
|  aggregations.TopLevelAggregate | 3403 | 3403 | 36 | 44 | 1134 |
|  aggregations.NestedAggregation | 16553 | 16553 | 76 | 91 | 2174 |
|  aggregations.AggregationWithWhere | 11979 | 11979 | 45 | 55 | 2174 |
|  aggregations.AggregationWhereWithinNestedRelationships | 22116987 | 22116987 | 3358 | 3367 | 2008534 |
|  aggregations.AggregationWhereWithinNestedConnections | 22116987 | 22116987 | 2900 | 2984 | 2008534 |
|  aggregations.NestedCountFromMovieToActors | 9734 | 9734 | 40 | 48 | 2174 |
|  aggregations.NestedCountFromActorsToMovie | 9937 | 9937 | 44 | 62 | 2174 |
|  aggregations.DeeplyNestedCount | 13070329 | 13070331 | 4420 | 4759 | 2008534 |
|  batch-create.BatchCreate | 4200 | 4200 | 153 | 140 | 600 |
|  batch-create.BatchCreateSmall | 77 | 77 | 54 | 74 | 11 |
|  connect.createAndConnect | 14424 | 14424 | 161 | 192 | 3003 |
|  connections.Connection | 14082 | 14082 | 59 | 68 | 2174 |
|  connections.NestedConnection | 45472 | 45470 | 113 | 132 | 4516 |
|  connections.ConnectionWithSort | 3284 | 3284 | 61 | 70 | 1040 |
|  connections.ConnectionWithSortAndCypher | 3284 | 3284 | 185 | 269 | 1040 |
|  create.SimpleMutation | 7 | 7 | 60 | 62 | 1 |
|  cypher-directive.TopLevelCypherDirective | 122100 | 122100 | 614 | 722 | 12241 |
|  cypher-directive.TopLevelCypherDirectiveWithColumnName | 153617 | 153617 | 168 | 166 | 12241 |
|  delete.SimpleDelete | 19401 | 19401 | 278 | 288 | 1040 |
|  delete.NestedDeleteInUpdate | 23202 | 23202 | 166 | 217 | 2040 |
|  2925.SingleRelationshipFilter | 6468 | 6468 | 133 | 52 | 1040 |
|  2925.NestedSingleRelationshipFilter | 22900 | 22900 | 126 | 127 | 2174 |
|  2925.SingleRelationshipRequiredFilter | 5201 | 5201 | 36 | 52 | 1040 |
|  2925.NestedSingleRelationshipRequiredFilter | 9361 | 9361 | 57 | 63 | 1040 |
|  query.SimpleQuery | 3121 | 3121 | 21 | 23 | 1040 |
|  query.SimpleQueryWithRelationship | 16162 | 16162 | 40 | 61 | 2174 |
|  query.QueryWhere | 9724 | 9703 | 38 | 40 | 2169 |
|  query.SimpleQueryWithNestedWhere | 9902 | 9881 | 49 | 58 | 2169 |
|  query.Nested | 10096041 | 10096041 | 10966 | 10778 | 2008534 |
|  query.NestedWithFilter | 10074401 | 10074401 | 11280 | 10546 | 2004000 |
|  query.OrFilterOnRelationships | 42792 | 43780 | 213 | 205 | 1742 |
|  query.OrFilterOnRelationshipsAndNested | 36227 | 37059 | 283 | 258 | 1742 |
|  query.QueryWithNestedIn | 13936 | 13383 | 103 | 68 | 1396 |
|  query.NestedConnectionWhere | 9834 | 9834 | 73 | 81 | 2174 |
|  query.DeeplyNestedConnectionWhere | 9911 | 9942 | 114 | 120 | 2174 |
|  query.DeeplyNestedWithRelationshipFilters | 19098 | 19200 | 134 | 293 | 1580 |
|  query.NestedWithRelationshipSingleFilters | 3881 | 3881 | 148 | 297 | 1134 |
|  query.Fulltext | 80 | 80 | 33 | 34 | 16 |
|  query.FulltextWithNestedQuery | 587 | 587 | 47 | 50 | 84 |
|  sorting.SortMultipleTypes | 3515 | 3515 | 65 | 85 | 1040 |
|  sorting.SortMultipleTypesWithCypherWithCypher | 1414 | 1471 | 155 | 259 | 1040 |
|  sorting.SortOnNestedFields | 14082 | 14082 | 55 | 71 | 2174 |
|  sorting.SortDeeplyNestedFields | 43198 | 43198 | 101 | 130 | 4516 |
|  sorting.SortWithTopLevelCypher | 3121 | 3121 | 104 | 181 | 1040 |
|  unions.SimpleUnionQuery | 321 | 321 | 71 | 64 | 35 |
|  unions.SimpleUnionQueryWithMissingFields | 293 | 293 | 62 | 73 | 35 |
|  unions.NestedUnion | 410637 | 410637 | 397 | 296 | 33033 |
|  unions.NestedUnionWithMissingFields | 384611 | 384611 | 273 | 289 | 33033 |
|  update.NestedUpdate | 16143 | 16143 | 95 | 256 | 2002 |

</details>
        
Old Schema Generation: 40.264s
Schema Generation: 39.569s
Old Subgraph Schema Generation: 48.333s
Subgraph Schema Generation: 49.854s

File: packages/graphql/src/classes/Node.ts [hunk 1]
```diff
@@ -81,22 +81,22 @@ type MutableField =
     | CustomScalarField
     | CustomEnumField
     | UnionField
-    | ObjectField
+
```
Comment (tbwiss) [inline packages/graphql/src/classes/Node.ts:84]: question: Shouldn't those lines be deleted? (there are several more in other files in this PR too)

**Review by tbwiss — DISMISSED** [review]
Toolbox stuff is 👍

File: packages/graphql/src/classes/Node.ts (hunk 1 above)
Comment (darrellwarde) [reply packages/graphql/src/classes/Node.ts:84]: Good spot, I think that's from where ESLint did some automatic fixing!

**Review by angrykoala — APPROVED** [review]

**Pull request merged**